pkgs/development/ocaml-modules/capnp/default.nix

```
@@ -0,0 +1,50 @@
+{ pkgs
+, lib
```

```suggestion
{ lib
```

This parameter is not used.

---

Sounds good! Removed.